### ISSUE TITLE
XEPs support page with version

### ISSUE THREAD

**Issue by Neustradamus** [miranda-ng/miranda-ng#1189]
On the XEPs support page, it is possible to add version of XEPs?
https://wiki.miranda-ng.org/index.php?title=Plugin:Jabber/en

Examples:
- https://metronome.im/supported-xeps-list
- https://trac.gajim.org/wiki/GajimXEPSupport
- https://github.com/jappix/jappix/blob/master/PROTOCOL.md

**Comment by dartraiden** [comment]
Yes, but I do not know how to determine which XEP version Miranda supports

**Comment by Neustradamus** [comment]
The solution is to see last XEPs version and look if all the code has been implemented, ... if not it is time to update...
If we do not take time to do, the client must be broken at a moment...
After, it will be easy to update the code...
It is same about RFCs... new versions have been released...

**Comment by sss123next** [comment]
i can say about my work:
https://xmpp.org/extensions/xep-0198.html - latest version  api 3, document 1.5.2 supported
https://xmpp.org/extensions/xep-0384.html - latest version 0.2 supported
https://xmpp.org/extensions/xep-0027.html - latest version 1.4 supported (via new_gpg plugin)
